packages/mongo-nav/src/project-nav/ProjectNav.tsx

```diff
@@ -198,6 +198,17 @@ const tooltipStyles = css`
   text-align: center;
 `;
 
+function displayProductName() {
```

Comment:
I'd just do this one and save it in a variable. No need to keep running this function on every render.

Reply:
secondTabName it is